specification/cognitiveservices/data-plane/EntitySearch/readme.md

```diff
@@ -18,13 +18,14 @@ These settings apply only when `--tag=release_1_0` is specified on the command l
 
 ``` yaml $(tag) == 'release_1_0'
 input-file: v1.0/EntitySearch.json
+add-credentials: y
```

Comment:
A. `add-credentials` is a boolean so this should look like `add-credentials:true`
B. Is there a reason why this flag is being set only for `release_1_0` tag? Looks like a code generation setting which should be independent of the tags; i.e. should be defined under the `# Code Generation` settings
An example [here](https://raw.githubusercontent.com/Azure/azure-rest-api-specs/8e7f780de76fa4005ca34898440f7810fa377fa9/specification/recoveryservicessiterecovery/resource-manager)/readme.md

Reply:
Sure thing(s)